source/NPC.h

```diff
@@ -109,8 +110,8 @@ class NPC {
 
 	// Create a copy of this NPC but with the fleets replaced by the actual
 	// ships they represent, wildcards in the conversation text replaced, etc.
-	NPC Instantiate(std::map<std::string, std::string> &subs, const System *origin, const System *destination,
-			int jumps, int64_t payload) const;
+	NPC Instantiate(std::map<std::string, std::string> &subs, const System *origin, const Planet *destinationPlanet,
```

Comment:
just thinking; this doesnt cause an issue if you want to go to a destination system with no planets?

Reply:
It doesn't appear to? Maybe there's some error or use case that I'm missing, but it seems to work fine.